service-worker.js

```diff
@@ -13,34 +13,11 @@ var offlineFundamentals = [
   //'service-worker.js',
   "manifest.webmanifest",
   "icon/android-chrome-192x192.png",
-  /*"node_modules/three/build/three.d.ts",
-  "node_modules/three/build/three.min.js",
-  "node_modules/three/examples/js/controls/DragControls.js",
-  "node_modules/three/examples/js/controls/OrbitControls.js",
-  "node_modules/three/examples/js/controls/TransformControls.js",
-  "node_modules/three/examples/js/exporters/STLExporter.js",
-  "node_modules/three/examples/js/exporters/OBJExporter.js",
-  "node_modules/controlkit/bin/controlKit.js",
-  "node_modules/jquery/dist/jquery.min.js",
-  "node_modules/opentype.js/dist/opentype.min.js",
-  "node_modules/golden-layout/dist/goldenlayout.min.js",
-  "node_modules/golden-layout/src/css/goldenlayout-base.css",
-  "node_modules/golden-layout/src/css/goldenlayout-dark-theme.css",
-  "node_modules/rawflate/rawdeflate.js",
-  "node_modules/rawflate/rawinflate.js",
-  "node_modules/opencascade.js/dist/opencascade.wasm.js",
-  "node_modules/opencascade.js/dist/opencascade.wasm.wasm",
-  "node_modules/monaco-editor/min/vs/editor/editor.main.css",
-  "node_modules/monaco-editor/min/vs/loader.js",
-  "node_modules/monaco-editor/min/vs/editor/editor.main.nls.js",
-  "node_modules/monaco-editor/min/vs/editor/editor.main.js",
-  "node_modules/monaco-editor/min/vs/basic-languages/javascript/javascript.js",
-  "node_modules/monaco-editor/min/vs/basic-languages/typescript/typescript.js",
-  "node_modules/monaco-editor/min/vs/language/typescript/tsMode.js",
-  "node_modules/monaco-editor/min/vs/language/typescript/tsWorker.js",
-  "node_modules/monaco-editor/min/vs/base/worker/workerMain.js",
-  "node_modules/monaco-editor/min/vs/base/browser/ui/codiconLabel/codicon/codicon.ttf",
-  "node_modules/opencascade.js/dist/oc.d.ts",
+  /*"static_node_modules/rawflate/rawdeflate.js",
+  "static_node_modules/rawflate/rawinflate.js",
+  "static_node_modules/opencascade.js/dist/opencascade.wasm.js",
+  "static_node_modules/opencascade.js/dist/opencascade.wasm.wasm",
+  "static_node_modules/opencascade.js/dist/oc.d.ts",
```

Comment:
Because I removed a number of the packages from static_node_modules (was node_modules), and now they are bundled, even though these are commented out I figured it was best to clean them up since they won't be cached in future.